File: JAVA/Basic/Square.java
//WAP to calc. area and circumference of circle
import java.util.Scanner;
class Square {
    public static void main(String[] as) {
        double a,ar,pr;
        Scanner ob = new Scanner(System.in);
        System.out.print("Enter the Length = ");
        a=ob.nextDouble();
        ar=a*a;
        pr=4*a;
        System.out.print("Area = "+ar+" Permimeter = "+pr);
        ob.close();
    }
}

File: C Programms/while loop/square.c
//WAP to print 1 4 9 16 25  .... n
#include<stdio.h>
void main()
{
    int n,i=1;
    printf("Enter the range = ");
    scanf("%d",&n);
    while(i<=n)
    {
        printf("%d ",i*i);
        i++;
    }
    printf(" = %d",(n*(n+1)*(2*n+1))/6);
}

